.github/workflows/deployBackendToInfraTest.yml

```diff
@@ -188,15 +188,15 @@ jobs:
           -var 'exporter_image_digest=${{ needs.build-exporter.outputs.image-digest }}' \
           -var 'frontend_image_digest=${{ needs.build-frontend.outputs.image-digest }}'
           data_server_url=$(terraform output data_server_url)
-          echo "data_server_url=$data_server_url" >> $GITHUB_OUTPUT
+          echo "data_server_url=$data_server_url" >> "$GITHUB_OUTPUT"
           ingestion_url=$(terraform output ingestion_url)
-          echo "ingestion_url=$ingestion_url" >> $GITHUB_OUTPUT
+          echo "ingestion_url=$ingestion_url" >> "$GITHUB_OUTPUT"
           gcs_to_bq_url=$(terraform output gcs_to_bq_url)
-          echo "gcs_to_bq_url=$gcs_to_bq_url" >> $GITHUB_OUTPUT
+          echo "gcs_to_bq_url=$gcs_to_bq_url" >> "$GITHUB_OUTPUT"
           exporter_url=$(terraform output exporter_url)
-          echo "exporter_url=$exporter_url" >> $GITHUB_OUTPUT
+          echo "exporter_url=$exporter_url" >> "$GITHUB_OUTPUT"
           frontend_url=$(terraform output frontend_url)
-          echo "frontend_url=$frontend_url" >> $GITHUB_OUTPUT
+          echo "frontend_url=$frontend_url" >> "$GITHUB_OUTPUT"
```

Comment:
https://github.com/SatcherInstitute/health-equity-tracker/actions/runs/7834887955/job/21379078888#step:4:223

The updated Super Linter now runs `shellcheck` on GitHub Actions' `run` command as well

Reply:
nice. thanks!